openapi/operation.go

```diff
@@ -45,20 +55,24 @@ func (o *Operation) getChildren() map[string]Traversable {
 func (o *Operation) setChild(i string, child Traversable) {
 	switch child.(type) {
 	case *Parameter:
-		// TODO: Handle this error
-		j, _ := strconv.Atoi(i)
 		param, _ := child.(*Parameter)
-		o.Parameters[j] = *param
+		o.Parameters = append(o.Parameters, *param)
```

Comment:
This will result in duplicates. What happens if my intention is to modify a child which was previously gotten using getChildren?

The original would now be left in the Parameters slice

Reply:
sorted